madara-prover-rpc-server/src/hints.rs

```diff
@@ -105,6 +105,26 @@ fn unimplemented_hint(
     Ok(())
 }
 
+/*
+Implements hint:
+%{
+    output_start = ids.output_ptr
+%}
+*/
+fn save_output_pointer_hint(
+    _vm: &mut VirtualMachine,
+    exec_scopes: &mut ExecutionScopes,
+    ids_data: &HashMap<String, HintReference>,
+    _ap_tracking: &ApTracking,
+    _constants: &HashMap<String, Felt252>,
+) -> Result<(), HintError> {
+    let output_ptr = ids_data.get("output_ptr")
+        .ok_or(HintError::CustomHint("output_ptr unavailable".to_owned().into_boxed_str()))?
```

Comment:
I think the correct error is `HintError::UnknownIdentifier` based on what I see in helpers like `get_relocatable_from_var_name`.

Reply:
410aaa1